bootstrap.py

```diff
@@ -82,6 +82,7 @@ def get_architecture():
 parser.add_option("-u", "--update", dest="update", action="store_true", help="Fully Updates the System")
 parser.add_option("-v", "--verbose", dest="verbose", action="store_true", help="Verbose output")
 parser.add_option("-f", "--force", dest="force", action="store_true", help="Force registration (will erase old katello and puppet certs)")
+parser.add_option("-d", "--add-domain", dest="add_domain", action="store_true", help="Automatically add the clients domain to Satellite6")
```

Comment:
s/Satellite6/Foreman/

also, I think I would only add the long `--add-domain` flag, `-d` sounds too much like debug or something and I think we do not need a short flag for everything-